frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -162,6 +167,59 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!isWindows && windowsTools) {
     dispatch(vmWizardInternalActions[InternalActionType.RemoveStorage](id, windowsTools.id));
   }
+
+  const iUserTemplates = iGetLoadedCommonData(state, id, VMWizardProps.userTemplates);
+  const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
+  const template = toShallowJS(iGetRelevantTemplate(iUserTemplates, iCommonTemplates, selectors));
+  const pvcName = getOperatingSystemDataVolumeAnnotation(template, selectors?.os);
+
+  dispatch(
+    vmWizardInternalActions[InternalActionType.UpdateVmSettingsField](
+      id,
+      VMSettingsField.CLONE_BASE_DISK_IMAGE,
+      {
+        value: !!pvcName,
+      },
+    ),
+  );
+};
+
+const mountBaseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
+  const state = getState();
+  if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.CLONE_BASE_DISK_IMAGE)) {
+    return;
+  }
+  const userTemplateName = iGetVmSettingValue(state, id, VMSettingsField.USER_TEMPLATE);
+  if (userTemplateName) {
```

Comment:
userTemplate do not support this? Even if - they should have unconditional say about the metadata (also it is missing in the prerequisites of this updater)

Reply:
user template sets the value of clone os base image to `false`,
in cases the user template sets the source to not disk, it will set the source accordingly
and in cases the user template set the source to disk, it will use the disk in the template (that may or may not be the same as the base os image)